book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
You need to add `:rentals` to your `attr_accessor` collection so that `Rental` class (/rental.rb line 12) be able to access that variable, read and set it to a new value. 👍🏽

Reply:
Ok.